### PR TITLE
News section update

### PR DESCRIPTION
**Pull request by AleksandraZhi**
Made these changes to the News section:

1. Moved all news to MongoDB under a new collection called "news"
2. News section became an _include_ instead of being hard-coded into index.pug
2. News section is being populated from MongoDB

news section

### CODE REVIEW

File: server/controllers/mainController.js [hunk 1]
```diff
@@ -5,10 +5,18 @@ const path = require('path')
...
 
-exports.mainPage = (req, res) => {
-	const copyrightYear = year()
-	res.render('../views/index', { copyrightYear })
+exports.mainPage = async (req, res) => {
+	try {
+		const totalNews = await News.countDocuments()
+		const allNewsArray = await News.find()
+		const copyrightYear = year()
+		console.log(allNewsArray, totalNews)
```
Comment (mforner13) [inline server/controllers/mainController.js]: Was this console.log used for debugging and can be removed? :)

File: server/views/includes/index-news.pug [hunk 2]
```diff
@@ -0,0 +1,9 @@
+.updates-scroll
+    - const arrayOfObjects = [...allNewsArray]
+    each value in arrayOfObjects
+        p.updates-date #{value.date}
+        p.updates-text #{value.text}
+        if (value.link !== '')
```
Comment (mforner13) [inline server/views/includes/index-news.pug]: Could you use the logical and operator (&&) here instead? :) (docs here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_AND)

**Review by mforner13 — COMMENTED** [review]
Looks good! Just two small comments :)

File: server/controllers/mainController.js (hunk 1 above)
Comment (AleksandraZhi) [reply server/controllers/mainController.js]: Thank you, yes, that was debugging. Sorry, missed this one.

File: server/views/includes/index-news.pug (hunk 2 above)
Comment (AleksandraZhi) [reply server/views/includes/index-news.pug]: Noted, updated. Thank you.

**Pull request merged**